### DISCUSSION TITLE
Use status colors on the main "Page" screen

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1301] [Feature Requests]
> Each status should have a Background Color, configured by the user, used in the pages list.
> 
> Just target the tr with class status-mystatus.
> 
> This is the Pages screen /wp-admin/edit.php?post_type=page ). 
> 
> Since it has alternating-row bg colors, a full background color might not be such a good idea. Instead, maybe a thin left-border on each page/row, based on status. 
> 
> I also added opacity to my custom status pages, as well as Draft and Pending. Helps a lot to easily see which pages (in the All list) are NOT published. Again, I can do this myself in css, but it might be a nice feature in your plugin.

**Comment by ghost** [comment]
The plugin, 'Post State Tags' is exactly what you're looking for, and recognizes PublishPress custom statuses, as well as others.

You can set the background color for each status, with an opacity level, and it also creates a left border to each TR with the set color. Once configured, be sure to enable the plugin in the 'Advanced Settings' tab within the plugin settings.

**Comment by stevejburge** [comment]
Thanks @maxlatitude 

@andergmartins Link is here: https://wordpress.org/plugins/post-state-tags/. Screenshot:

![screenshot-1](https://user-images.githubusercontent.com/3868028/72553090-22ce8180-3866-11ea-8f13-43f256c17fba.png)

**Comment by stevejburge** [comment]
One more request for this from @heyjoecampbell ;)

**Comment by stevejburge** [comment]
Here's a slightly different approach to the same thing: https://wordpress.org/plugins/post-status-indicator/
